### ISSUE TITLE
Feature request: Include preceeding code blocks in paragraph highlighting

### ISSUE THREAD

**Issue by BillyONeal** [Eelis/cxxdraft-htmlgen#32]
Consider: http://eel.is/c++draft/util.smartptr#shared.const-18

It would be nice if the:

```
shared_ptr(const shared_ptr& r) noexcept;
template <class Y> shared_ptr(const shared_ptr<Y>& r) noexcept;
```

were included in the green background that's being referred to. (Or if there were a way to form a link which would highlight that code block)

**Comment by Eelis** [comment]
Fixed by 142421a84f44dcf3488d9fbc4b40749c6e823a04. Moving the mouse over the left margin of an itemdecl now reveals a link that selects the itemdecl ([example](http://eel.is/c++draft/util.smartptr.shared.const#itemdecl:5)).

**Comment by BillyONeal** [comment]
YAAAAAAAAAAAAAY  :D :D

Thanks!
